docs/csharp/language-reference/preprocessor-directives.md

```diff
@@ -184,7 +184,7 @@ You use `#define` to define a symbol. When you use the symbol as the expression
  ```
 
 > [!NOTE]
-> The `#define` directive cannot be used to declare constant values as is typically done in C and C++. Constants in C# are best defined as static members of a class or struct. If you have several such constants, consider creating a separate "Constants" class to hold them.
+> In C#, primitive constants should be defined using the `const` keyword. The `#define` directive cannot be used to declare constant values as is typically done in C and C++. You can also define constants in C# as static members of a class or struct, but be aware that static members are resolved at runtime, and thus not necessarily read-only. If you have several such constants, consider creating a separate "Constants" class to hold them.
```

Comment:
The change here could be misinterpreted by some readers. Members declared with the `const` keyword are `static` members. The distinction between using the `const` keyword and the `static` keyword is that `const` members are readonly. How about this wording instead:

```suggestion
> In C#, primitive constants should be defined using the [`const`](keywords/const.md) keyword. A `const` declaration creates a `static` member that can't be modified at runtime. The `#define` directive can't be used to declare constant values as is typically done in C and C++. If you have several such constants, consider creating a separate "Constants" class to hold them.
```